packages/blocks/src/components/format-quick-bar/config.ts

```diff
@@ -23,37 +22,90 @@ import {
   UnderlineIcon,
 } from './icons.js';
 
-export const paragraphButtons = [
+export const paragraphConfig = [
   {
     flavour: 'affine:paragraph',
     type: 'text',
     name: 'Text',
+    hotkey: 'command+option+0,ctrl+shift+0',
     icon: TextIcon,
   },
-  { flavour: 'affine:paragraph', type: 'h1', name: 'Heading 1', icon: H1Icon },
-  { flavour: 'affine:paragraph', type: 'h2', name: 'Heading 2', icon: H2Icon },
-  { flavour: 'affine:paragraph', type: 'h3', name: 'Heading 3', icon: H3Icon },
-  { flavour: 'affine:paragraph', type: 'h4', name: 'Heading 4', icon: H4Icon },
-  { flavour: 'affine:paragraph', type: 'h5', name: 'Heading 5', icon: H5Icon },
-  { flavour: 'affine:paragraph', type: 'h6', name: 'Heading 6', icon: H6Icon },
+  {
+    flavour: 'affine:paragraph',
+    type: 'h1',
+    name: 'Heading 1',
+    hotkey: 'command+option+1,ctrl+shift+1',
```

Comment:
It seems that this `hotkey` config fields are defined in the FormatQuickBar, will it make more sense to make them a standalone configuration module global level, e.g., next to the the `bind-hotkey.ts`?